tidb-cloud/dev-guide-bi-looker-studio.md

```diff
@@ -25,7 +25,7 @@ To complete this tutorial, you need:
 
 **If you don't have a TiDB Serverless cluster, you can create one as follows:**
 
-- [Create a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-tidb-serverless-cluster)
+- [Create a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-tidb-cloud-serverless-cluster)
```

Comment:
Updating the link in this file is to pass the TiDB Cloud link check, to avoid broken anchor links.